payments/enums.py

```diff
@@ -70,12 +70,19 @@ class OrderStatus(TextChoices):
     ERROR = "error", _("Error")
     PAID = "paid", _("Paid")
     PAID_MANUALLY = "paid_man", _("Paid manually")  # max 9 chars
+    REFUNDED = "refunded", _("Refunded")
```

Comment:
> _When the berth is refunded, is it considered free? _

I think it shouldn't. As we've discussed before, the berth is only released until the lease is terminated. So it could be the case that the invoice was incorrect and the customer will get a new invoice. So I think they should keep the berth until manually terminated. (I asked Joonas to confirm this tho)

Reply:
<img width="746" alt="image" src="https://user-images.githubusercontent.com/15201480/110761957-fab79400-8258-11eb-8482-9560f51838c3.png">

Reply:
EDIT: I discussed with Joonas and we agred that, if the refund is a full refund, it will ALWAYS terminate the lease and release the berth.

Reply:
Updated diagram:
<img width="941" alt="image" src="https://user-images.githubusercontent.com/15201480/110921002-b5fa2e80-8326-11eb-938b-e0e7680181a4.png">